modules/ve-cx/init/ve.init.mw.CXTarget.js

```
@@ -530,6 +530,8 @@ ve.init.mw.CXTarget.prototype.onSurfaceReady = function () {
 
 	if ( this.translation.hasTranslatedSections() ) {
 		this.targetSurface.$element.addClass( 've-ui-cxTargetSurface--non-empty' );
+	} else if (this.translation.hasTranslationConflict()) {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)